lua/vgit/settings/hls.lua

```diff
@@ -0,0 +1,26 @@
+local Config = require('vgit.core.Config')
+
+return Config:new({
+  GitBackgroundPrimary = 'NormalFloat',
+  GitBackgroundSecondary = {},
+  GitBorder = 'LineNr',
+  GitLineNumber = 'LineNr',
+  GitViewSignAdd = 'DiffAdd',
+  GitViewSignDelete = 'DiffDelete',
+  GitComment = 'Comment',
+  GitViewWordAdd = {
+    bg = '#5d7a22',
+  },
+  GitViewWordDelete = {
+    bg = '#960f3d',
+  },
+  GitSignAdd = {
+    fg = '#d7ffaf',
+  },
+  GitSignChange = {
+    fg = '#7AA6DA',
+  },
+  GitSignDelete = {
+    fg = '#e95678',
+  },
```

Comment:
Maybe use `GitSigns*` (plural)?
This is how it named in [gitsigns](https://github.com/lewis6991/gitsigns.nvim/blob/main/lua/gitsigns/highlight.lua#L27).
```suggestion
  GitSignsAdd = {
    fg = '#d7ffaf',
  },
  GitSignsChange = {
    fg = '#7AA6DA',
  },
  GitSignsDelete = {
    fg = '#e95678',
  },
```